src/pages/new/index.tsx

```diff
@@ -77,11 +77,11 @@ const CourseFeatureCard = ({resource, className}: any) => {
               <p className="text-sm mt-4">{description}</p>
             </div>
           </div>
-          <img
+          {/* <img
             className="absolute top-0 left-0 z-0 w-full"
             src={featureCardBackground}
             alt=""
-          />
+          /> */}
```

Comment:
Remove the feature card background. Less noise. Focus on the hero banner. 

![image](https://user-images.githubusercontent.com/57044804/118532693-18432700-b6fc-11eb-8eba-5b02191c62a3.png)

vs.

![image](https://user-images.githubusercontent.com/57044804/118532768-2d1fba80-b6fc-11eb-9b45-d07b4b281a40.png)